markdown/developers/guidelines/requirements/single_command.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Single command
+subtitle: Pipelines should run in a single command.
+menu:
+  main:
+    weight: 110
+---
+
+Every nf-core pipeline repository should contain a single pipeline.
+That is, there should be a `main.nf` file that is the single way to launch a pipeline.
+
+- It is ok to have multiple branches within the pipeline, selectable with configuration options.
```

Comment:
```suggestion
- It is ok to have multiple 'tracks' within the pipeline, selectable with configuration options.
```

Given we've discussed branches extensively in the context of git(hub) in the rest of the document, I misread this at first